.taskcluster.yml

```diff
@@ -57,7 +57,7 @@ tasks:
             owner: ${owner}
             source: ${event.repository.clone_url}
           payload:
-            image: webplatformtests/wpt:0.54
+            image: webplatformtests/wpt:0.53
```

Comment:
Reverting this might cause problems, since @jgraham says the version on the branch will be used. Some PRs were already broken when landing https://github.com/web-platform-tests/wpt/pull/40130 and required a rebase. The same thing ought to happen again if we land this, since the docker image won't have virtualenv, but the code being run requires it.